cnxepub/tests/test_formatters.py

```diff
@@ -624,20 +624,23 @@ def setUp(self):
 
         metadata = self.base_metadata.copy()
         contents = io.BytesIO(u"""\
+<div>
```

Comment:
What would happen without adding this line and the closing tag? I assume they are relevant so I'm guessing that the test might fail because it'd be considered broken html, xhtml, or xml. Maybe? And the function being deleted in this PR that "sanitizes" XML was previously making the tests pass?

Reply:
Correct - with this PR, we no longer support (and never really used) sending fragments as document content.

Reply:
The bulk of these are tests setting up content to test something other than setting up content: they just took advantage of support for HTML fragments (rather than a valid XHTML tree). The code would wrap them in a div, as part of the fragment processing, so I just did that in the tests directly.